developer/js/source/lexical-model-compiler/build-trie.ts

```diff
@@ -4,7 +4,7 @@ import { readFileSync } from "fs";
  * A word list is an array of pairs: the concrete word form itself, followed by
```

Comment:
Is word list now an Object?

Reply:
The terminology gets tricky here.  The new `WordList` definition technically is rooted in a JavaScript object, yes.  However, the technical term for this pattern is [_associative array_]().

That said, even if we kept the "array" term, it's no longer an array of pairs.  It's more like a Java `HashMap`.

Reply:
> Java

Well, actually... :rofl: